Vue3

kylewalow opened the pull request.

kylewalow committed:
update plugin entry for vue3

kylewalow committed:
update package version

kylewalow committed:
Merge branch 'main' into vue3

strebl commented on package.json:
```diff
@@ -48,9 +48,9 @@
     "standard-version": "latest",
-    "vue": "^2.6.11",
+    "vue": "^3.2.45",
     "vue-template-compiler": "^2.6.11"
```
isch das die aktuellsti und korrekti version?

strebl commented on src/entry.js:
```diff
@@ -1,41 +1,8 @@
-    GlobalVue.use(plugin)
-  }
+export default {
+  install: (app, options) => {
```
mach doch do die aktuellst syntax:

```js
install(app, options) {
  ...
}
```

strebl replied:
mach doch do die aktuellst syntax:

```js
install(app, options) {
  ...
}
```

strebl commented on package.json:
```diff
@@ -30,7 +30,7 @@
     "@rollup/plugin-commonjs": "^11.1.0",
     "@rollup/plugin-replace": "^2.3.2",
-    "@teamnovu/vue-cloudinary-image": "0.0.11",
+    "@teamnovu/vue-cloudinary-image": "2.0.0",
```
sötts do nid v3 me?

kylewalow committed:
1.0.1-dev.2

kylewalow committed:
fix main export in package.json and add pkg.exports

kylewalow committed:
1.0.1

kylewalow committed:
fix copy pasta issue in package.json

kylewalow committed:
1.0.2

kylewalow committed:
fix: peer dependency to vue 3

kylewalow committed:
1.0.3

arcs- commented:
@strebl @Kylewalow hend ihr das no ufm schirm?

arcs- committed:
fix  #3: set alt fallback

arcs- committed:
Merge branch 'vue3' of github.com:teamnovu/vue-cloudinary-image-statamic into vue3

arcs- committed:
chore: new version

strebl committed:
DX improvements (#4)

* feat: add i18n support

* feat: don't report missing property

* fix: ensure that empty alt text is handled correctly

Currently a null alt text from Statamic is rendered as <img ... alt
...>. This makes it so that the alt attribute is only set if the alt
text contains actual content.

---------

Co-authored-by: Oguz Han Asnaz <oguzhan.asnaz@novu.ch>

strebl committed:
chore: release v1.0.4

strebl committed:
chore(release): 1.0.5

strebl committed:
chore(release): 1.0.6

strebl committed:
feat: set object position by default

strebl committed:
chore(release): 1.1.0
